cmd/chisel/cmd_cut.go

```diff
@@ -57,6 +58,14 @@ func (cmd *cmdCut) Execute(args []string) error {
 		}
 		sliceKeys[i] = sliceKey
 	}
+	arch := cmd.Arch
+	if arch == "" {
+		darch, err := deb.InferArch()
```

Comment:
NOTE: this is returning the `debArch` field of `archPair` is that the correct one to use? maybe in the future we should do an internal refactor for `goArch` and `debArch` to be enums to make it clear which one we're using where.. 🤔